src/index.html

```diff
@@ -7,6 +7,73 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="calendar">
+      <div class="calendar__field">
+        <div class="calendar__cell calendar__cell--shift"></div>
```

Comment:
```suggestion
        <div class="calendar__cell"></div>
```
do not extra classes